### PR TITLE
Added option and example for access transformers

### PR DESCRIPTION
**Pull request by SinTh0r4s**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -154,6 +154,7 @@ shadowJar {
         from refMap
         manifest {
             attributes([
+                    "FMLAT"                      : accessTransformersFile.toString(),
```
Comment (SinTh0r4s) [inline build.gradle:157]: How shall we add it conditionally? Only when `accessTransformersFile.isEmpty() == false`?

File: src/main/java/com/myname/mymodid/CommonProxy.java [hunk 2]
```diff
@@ -20,7 +23,14 @@ public void init(FMLInitializationEvent event) {
...
     // postInit "Handle interaction with other mods, complete your setup based on this."
     public void postInit(FMLPostInitializationEvent event) {
+        // Access originally package-private subclass
+        GuiScreenBook.NextPageButton button = null;
 
+        // Access originally private field
+        Material air = Blocks.air.blockMaterial;
+
+        // Access originally protected method
+        Blocks.air.disableStats();
```
Comment (SinTh0r4s) [inline src/main/java/com/myname/mymodid/CommonProxy.java:33]: Do we have them enabled by default? Alternatively, we could comment them out and deactivate the option in `gradle.properties`

File: build.gradle (hunk 1 above)
Comment (TheElan) [reply build.gradle:157]: I am not sure how groovy arrays are handled, but you could probably define a variable like:
```
def manifestParams = [
"pram1WhichAlwaysPresent": "$param1Value",
"param2WhichAlwaysPresnet": "$param2Value",
...
]

and then just if and `+= ["...": "..."]` or `+= "...": "..."` or what ever syntax this has or migrate to kotlin and have clearly defined syntax of summing sets/maps ;p

File: src/main/java/com/myname/mymodid/CommonProxy.java (hunk 2 above)
Comment (TheElan) [reply src/main/java/com/myname/mymodid/CommonProxy.java:33]: I would rather go with creating a separate page explaining how reflections are activated and used in 1.7.10 with code snippet(s)

**Pull request merged**